tests/e2e/demo-homepages.spec.ts

```diff
@@ -1,13 +1,14 @@
+import { SERVERS, getDevelopmentEndpointUri } from '@lmc-eu/spirit-common/constants/servers';
 import { expect, test } from '@playwright/test';
 
 test.describe('Demo Homepages', () => {
   const demos = [
     {
-      url: 'https://spirit-design-system-demo.netlify.app/',
+      url: process.env.CI ? SERVERS.PRODUCTION.web : getDevelopmentEndpointUri('web'),
```

Comment:
Will there be any more adjustments based on the `ci` flag? Because something like `urls = [ local | production ]` seems to me to better describe what it does now.

Reply:
Would you rather prefer something like `process.env.NODE_ENV` be used instead of `process.env.CI`?

The `NODE_ENV` has values like `production`, `development`, and so on.

Reply:
I have set up the environments and used `NODE_ENV` instead of `CI`. This could leave us more space to define environments against which we can run our tests.